gefserver/pier/internal/dckr/dckr.go

```diff
@@ -647,6 +658,38 @@ func (c Client) WaitContainerOrSwarmService(id string, removeOnExit bool) (int,
 	}
 }
 
+// ForceRemoveContainerOrSwarmService stops and removes a running container or a swarm service
+func (c Client) ForceRemoveContainerOrSwarmService(id string) error {
+	// Inspection will fail, if the container does not exist
+	_, err := c.c.InspectContainer(id)
```

Comment:
I'm not sure what's the purpose of this check. Must it be done in swarm mode too?